### PR TITLE
Colophon update

### PR DESCRIPTION
**Pull request by Nitin-TELUS**
Update colophon details

### CODE REVIEW

File: .colophon.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - type: Architect
+    name: Matthew Cantelon
+    email: matthew.cantelon@telus.com
+    github: mattcan
+
+references:
+  - title: Source Code
+    uri: https://github.com/telus/telus-colophon-app
+  - title: Homepage
+    uri: https://colophon.id
```
Comment (maumercado) [inline .colophon.yml]:
```suggestion
    uri: https://colophon.telus.digital
```

File: .colophon.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+references:
+  - title: Source Code
+    uri: https://github.com/telus/telus-colophon-app
+  - title: Homepage
+    uri: https://colophon.telus.digital
+
+environments:
+  - title: NPM Package
+    uri: https://www.npmjs.com/package/@telus/telus-colophon-app
```
Comment (pdufault) [inline .colophon.yml]: I don't see a package published here: https://www.npmjs.com/package/@telus/telus-colophon-app

Comment (maumercado) [reply .colophon.yml]: 🤔 good observation

Comment (maumercado) [inline .colophon.yml]:
```suggestion
    uri: https://www.npmjs.com/package/@telus/colophon-app
```

Comment (maumercado) [reply .colophon.yml]: That is the actual url of the published package

Comment (Nitin-TELUS) [reply .colophon.yml]: @maumercado, update the url

**Comment by kehkansha** [comment]
I have merge this PR with [https://github.com/telus/telus-colophon-app/pull/13](url)

**Pull request closed without merging**